### PR TITLE
Adds mobility

### PR DESCRIPTION
**Pull request by ignasi35**
References #59
Built on top of #98

### CODE REVIEW

File: docs-source/docs/modules/concepts/pages/mobility.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+:toclevels: 2
+
+include::ROOT:partial$include.adoc[]
+
+
+With the advent of cloud computing, virtualization, and Docker containers, we have a lot of power at our disposal to efficiently man‐ age hardware resources. The problem is that none of this matters if our Microservices and its underlying platform cannot make efficient use of it. What we need are services that are mobile, allowing them to be elastic.
+
+We have talked about asynchronous message-passing, and that it provides decoupling in time and space. The latter, decoupling in space, is what we call link:{glossary-location-transparency}[Location Transparency], the ability to, at run‐ time, dynamically scale the Microservice—either on multiple cores or on multiple nodes—without changing the code. This is service distribution that enables elasticity and mobility; it is needed to take full advantage of cloud computing and its pay-as-you-go models.
+
+For a service to become location transparent it needs to be addressa‐ ble. But what does that really mean?
```
Comment (patriknw) [inline docs-source/docs/modules/concepts/pages/mobility.adoc:13]: addressa‐ ble looks funny?

File: docs-source/docs/modules/concepts/pages/mobility.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+First, addresses need to be stable in the sense that they can be used to refer to the service indefinitely, regardless of where it is currently located. This should hold true if the service is running, has been stopped, is suspended, is being upgraded, has crashed, and so on. The address should always work (Figure 2-8). This means a client can always send messages to an address. In practice they might sometimes be queued up, resubmitted, delegated, logged, or sent to a dead letter queue.
+
+Second, an address needs to be virtual in the sense that it can, and often does, represent not just one, but a whole set of runtime instan‐ ces that together defines the service. Reasons this can be advanta‐ geous include:
```
Comment (patriknw) [inline docs-source/docs/modules/concepts/pages/mobility.adoc:17]: instan‐ ces ?

**Review by patriknw — APPROVED** [review]
LGTM, after minor...

File: docs-source/docs/modules/concepts/pages/mobility.adoc (hunk 1 above)
Comment (ignasi35) [reply docs-source/docs/modules/concepts/pages/mobility.adoc:13]: Yeah, that's result of copy/pasting from the PDF. I'll do a general pass with `s/- //` (see `run- time` in the previous paragraph aswell). Good catch.

Comment (ignasi35) [reply docs-source/docs/modules/concepts/pages/mobility.adoc:13]: For the record, I'll use `runtime` over `run-time` following https://english.stackexchange.com/a/67014

**Pull request merged**